src/main/java/models/Player.java

```diff
@@ -0,0 +1,18 @@
+package models;
+
+public class Player {
+
+    private String name;
+    private int score;
+
+    public Player(String name, int score){
+        this.name = name;
+        this.score = score;
+    }
+
+    public int getScore() {
+        return score;
+    }
+
+    public String getName() { return name; }
```

Comment:
I'd change `getName()` to multiple lines as well in order to stay consistent with the coding style!

Reply:
+1